packages/admin/admin/src/snackbar/UndoSnackbar.tsx

```diff
@@ -6,13 +6,13 @@ import * as UUID from "uuid";
 
 import { useSnackbarApi } from "./SnackbarProvider";
 
-export interface UndoSnackbarProps<Payload extends unknown> extends Omit<SnackbarProps, "action"> {
+export interface UndoSnackbarProps<Payload> extends Omit<SnackbarProps, "action"> {
```

Comment:
Developers are required to specify the snackbar's payload with this change. Is this what we want? May we should just type it as `Payload = any` instead?

Reply:
I didn't changed that, this was autofixed by eslint. this would be something for later?

Reply:
I guess that TS is still able to infer that type from the provided props. Otherwise ESLint wouldn't have fixed that.